src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```
@@ -137,6 +137,16 @@ exclude-jmod-section.usage=\
 \                            Specify a JMOD section to exclude.\n\
 \                            Where <section-name> is \"man\" or \"headers\".
 
+generate-cds-archive.argument=
+
+generate-cds-archive.description=\
+CDS plugin: generate cds archives (classes.jsa, classes_nocoops.jsa).\n\
+Applicable to JDK images built with the CDS feature.
+
+generate-cds-archive.usage=\
+\  --generate-cds-archive    Generate CDS archives (classes.jsa, classes_nocoops.jsa).\n\
+\                            Applicable to JDK images built with the CDS feature.
```

```suggestion
\  --generate-cds-archive    Generate CDS archives if the runtime image supports CDS feature.\n\
```

Does this match what you want to say?

---

Yes.